recipes/test_resource.rb

```diff
@@ -0,0 +1,10 @@
+# This is a helper recipe executing the default action of the resource
```

Comment:
Nitpick: not sure this should be under `recipes` and not under `resources`.

Reply:
For the time being I would keep this at the resource level for clarity.